lib/src/rendering/overflow_view.dart

```diff
@@ -18,26 +18,32 @@ class RenderOverflowView extends RenderBox
         ContainerRenderObjectMixin<RenderBox, OverflowViewParentData>,
         RenderBoxContainerDefaultsMixin<RenderBox, OverflowViewParentData> {
   RenderOverflowView({
-    List<RenderBox> children,
-    Axis direction,
-    double spacing,
-    OverflowViewLayoutBehavior layoutBehavior,
-  })  : assert(direction != null),
-        assert(spacing != null &&
-            spacing > double.negativeInfinity &&
+    List<RenderBox>? children,
+    required Axis direction,
+    required double spacing,
+    required OverflowViewLayoutBehavior layoutBehavior,
+  })  : assert(spacing > double.negativeInfinity &&
             spacing < double.infinity),
-        assert(layoutBehavior != null),
         _direction = direction,
         _spacing = spacing,
         _layoutBehavior = layoutBehavior,
         _isHorizontal = direction == Axis.horizontal {
     addAll(children);
   }
 
-  Axis get direction => _direction;
   Axis _direction;
+
+  double _spacing;
```

Comment:
I prefer that we keep the fields where there was. This is the convention used in the Flutter framework and I want to keep it if you don't mind.